src/content/en/updates/2019/01/rtcquictransport-api.md

```diff
@@ -0,0 +1,246 @@
+alt="RTCQuicTransport diagram showing architecture of API">
+
+#### Client perspective:
+
+```
+const iceTransport = new RTCIceTransport();
+const quicTransport = new RTCQuicTransport(iceTransport);
+// Signal parameters, key and candidates.
+signalingChannel.send({
+  iceParams: iceTransport.getLocalParameters(),
+  quicKey: quicTransport.getKey(),
+});
+iceTransport.onicecandidate = e => {
+  if (e.candidate) {
+    signalingChannel.send({candidate: e.candidate} );
+  }
+}
+
+// When remote parameters are signaled, start connection.
+signalingChannel.onMessage = async ({iceParams, candidate}) => {
+  if (iceParams) {
+    iceTransport.start(iceParams);
+    quicTransport.connect();
+  } else if (candidate) {
+    iceTransport.addRemoteCandidate(candidate);
+  }
+};
+```
+
+#### Server perspective:
+
+```
+const iceTransport = new RTCIceTransport();
+const quicTransport = new RTCQuicTransport(iceTransport);
+// Signal parameters, key and candidates.
+signalingChannel.send({
+  iceParams: iceTransport.getLocalParameters(),
+});
+iceTransport.onicecandidate = e => {
+  if (e.candidate) {
+    signalingChannel.send({candidate: e.candidate});
+  }
+}
+
+// When remote parameters are signaled, start connection.
+signalingChannel.onMessage = async ({iceParams, quicKey, candidate}) => {
+  if (iceParams && quicKey) {
+    iceTransport.start(iceParams);
+    quicTransport.listen(quicKey);
+  } else if (candidate) {
+    iceTransport.addRemoteCandidate(candidate);
+  }
+};
+```
+
+### Data Transfer
+
+Data transfer can be achieved  using the RTCQuicStream APIs for reading and
+writing:
+```
+RTCQuicStreamReadResult readInto(Uint8Array data);
+void write(RTCQuicStreamWriteParameters data);
+Promise<void> waitForWriteBufferedAmountBelow(unsigned long amount);
+Promise<void> waitForReadable(unsigned long amount);
+```
+
+#### Buffering
+
+The promises returned by the waitFor\* methods allow buffering data when
+JavaScript is busy. Back pressure is applied to the send side when the
+read buffer becomes full on the receive side. The send side has a write
+buffer that can fill when back pressure has been applied, and therefore
+the write side has a waitFor* method as well to allow waiting for room in
+the buffer to write. More information on writing/reading data can be found
+in the further developer documentation.
+
+#### Unordered/Unreliable Delivery {: #unordered }
+
+While an RTCQuicStream only supports sending data reliably and in order,
+unreliable/unordered delivery can be achieved through other means. For
+unordered delivery, one can send small chunks of data on separate streams
+because data is not ordered between streams. For unreliable delivery, one
+can send small chunks of data with finish set to true, followed by calling
+reset() on the stream after a timeout. The timeout should be dependent on
+how many retransmissions are desired before dropping the data.
+
+## When?
+
+The origin trial will start in the M73 version, and will be available
```

Comment:
We try to avoid using "m73" and instead prefer to call things "Chrome 73", external developers don't think in terms of milestone.

Reply:
Thanks for the catch.